### ISSUE TITLE
Unable to detect in-gmae GUI offset!

### ISSUE THREAD

**Issue by qqq123400** [NatroTeam/NatroMacro#418]
### Have you searched the existing bug reports?

Yes

### What happened?

![Snipaste_2024-04-05_06-24-03](https://github.com/NatroTeam/NatroMacro/assets/21179287/8f263bd3-1403-4a2e-ac4d-54c5d589741f)

### Steps to reproduce

1.
2.
3.

### What would you expect to happen?

Already tried many times
Discord troubleshooting and setting adjustments cannot be effectively solved.
There is a very small chance that it will be normal, but after a day or so, the same prompt will be displayed again, making it impossible to detect motion offset stably.

Version 0.9.7 will not have this problem.
But version 1.0.0.0 and other versions still have this problem
Is there a solution? Or more or in-depth adjustments?
I am willing to learn and operate

### Natro Macro version

1.0.0

### What is your Discord username?

Shark_Tw01

**Comment by ninjubaer** [comment]
Hello @qqq123400 
The warning "Unable to detect in-game Gui offset" appears when the macro for some reason is unable to detect the pollen text at the top of the screen.
There is a [guide](https://discord.com/channels/1012610056921038868/1190624759277441024) on how to fix this issue on our [discord server](https://discord.gg/natromacro)

**Comment by qqq123400** [comment]
I finally solved this problem
Because my own Roblox was not [English] before entering the game.
When I select the language before entering [Bee Swarm Simulator] and change it to English, I can detect top pollen.
There will be no GUI detection issues.

I originally thought I only needed to set the [in-game] language
I didn’t expect that Roblox’s language before entering the game would also need to be adjusted.

This message is for reference by those who have encountered the same problem.
